src/ray/object_manager/plasma/plasma_allocator.cc

```diff
@@ -89,10 +89,12 @@ absl::optional<Allocation> PlasmaAllocator::Allocate(size_t bytes) {
     return absl::nullopt;
   }
   allocated_ += bytes;
-  return BuildAllocation(mem, bytes);
+  return BuildAllocation(mem, bytes, /* is_fallback_allocated */ false);
 }
```

Comment:
I thought that previously we decided plasma should only record Unsealed objects, and LocalObjectManager should record locations for the rest (Spilled, InMem)?

Reply:
Yeah - after chatting with Stephanie, the InMem info on LocalObjectManager is incomplete: LocalObjectManager only sees the Primary Copy related usage (those pinned). The total InMem needs to be reported from the plasma. 

So the current proposed split will be: 
- LocalObjectManager: Spill bytes
- Plasma store: Fallback + InMemory

Reply:
Ok, that makes sense.